### PR TITLE
chore: separate vars by distro, fix FD plugin integration

### PR DESCRIPTION
**Pull request by adf-patrickha**
* Fix FD Plugin integration to support multiple different distributions and different kind of plugins.
* Separate vars by distribution according to Adfinis Ansible Guide

Refs: https://adfinis.github.io/ansible-guide/roles_collections.html#variables

### CODE REVIEW

**Comment by adf-patrickha** [comment]
Back to draft. The rendered FD config (`/etc/bareos/bareos-fd.d/client/myself.conf`) is not rendered correctly.

**Comment by adf-patrickha** [comment]
OK, should finally be ready to review. @fadnincx can you have a look?

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -16,3 +16,20 @@
...
+    # have to be specified in converge.yml: `bareos_fd_plugins`
+    - name: Check if plugin packages were installed
+      ansible.builtin.package:
+        name:
+          - bareos-filedaemon-mariabackup-python-plugin
+          - bareos-filedaemon-postgresql-python-plugin
+          # TODO disabled as package `bareos-filedaemon-ldap-python-plugin`
+          # has broken dependencies on Debian (python-ldap instead of python3-ldap)
+          # - bareos-filedaemon-ldap-python-plugin
+        state: present
```
[the remark this answers is not in the document]
Comment (adf-patrickha) [reply molecule/default/verify.yml:29]: there is no dedicated package for the MySQL plugin. it's just using the "python-mysql" library which differs between distros, so I did not add it here.

File: vars/RedHat.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+bareos_fd_plugin_dir: "/usr/lib64/bareos/plugins"
+bareos_fd_plugin_name: python3
+bareos_fd_plugin_list:
+  - name: mariabackup
+    packages:
+      - bareos-filedaemon-mariabackup-python-plugin
+
+  # TODO RHEL offers no python-mysqlclient package per default.
+  # Check if PyMySQL is working with the plugin
```
[the remark this answers is not in the document]
Comment (adf-patrickha) [reply vars/RedHat.yml:15]: cannot be removed for now. I'll leave it in and github-actions created a related issue here: https://github.com/adfinis/ansible-role-bareos_fd/issues/4

**Review by fadnincx — APPROVED** [review]
LGTM

**Pull request merged**